proceeding/tables/catalog_table.tex

```diff
@@ -0,0 +1,21 @@
+% Table
+\begin{table}[h]
+
+\caption{Catalog information.}
+\label{tab:a}
+\tabcolsep7pt\begin{tabular}{ || c|ccc ||}
+\hline
+% \tch{1}{c}{b}{Catalog\\ name} & \tch{1}{c}{b}{Number\\ of sources}  & \tch{1}{c}{b}{Update}  & \tch{1}{c|}{b}{Description}  \\
+% & \tch{1}{c}{b}{Single\\ outlet}  & \tch{1}{c}{b}{Small\\ multiple\tabnoteref{t1n1}}  & \tch{1}{c}{b}{Large\\ multiple}  & \tch{1}{c}{b}{Total}   \\
+
+
+\textbf{Catalog name} & \textbf{Number of sources} & \textbf{Update} & \textbf{Description} \\ \hline
```

Comment:
Shorter titles to gain some space for description?
- Catalog name -> Catalog
- Number of sources -> Sources
You can still mention that "number of sources" is given in the caption, so that it's clear.